src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/libraries",
+    "description":"Google's fully managed, petabyte scale, serverless data warehouse for analytics",
+    "transport": "http",
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/bigquery.svg",
+    "clients": [
+      {
+      "language":"java",
+      "site":"https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/0.22.0/apidocs/com/google/cloud/bigquery/package-summary.html",
+      "version" : "0.22.0",
+      "infotip":"The com.google.cloud.bigquery packages",
+      "status" : "beta",  
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigquery",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-bigquery",
+         "version" : "0.22.0-beta"
+      },
+      "compatible" : [
+          "Android",
```

Comment:
the Android and AES-7 elements are wrong. They're explicitly unsupported for all google-cloud-java libraries.

more generally AES-8 and 'Java 7' are supposed to be equivalent by design.  i.e. AES-8 is not a 'special' Java environment. It's standard Java and we don't need to be explicit about it's compatibility IMO.